app/component/header.tsx

```diff
@@ -0,0 +1,26 @@
+import ArrowLeft from "react:~assets/arrowLeft.svg"
+import { Link } from "wouter"
+
+import { Path } from "~app/path"
+
+type HeaderProps = {
+  title: string
+  children?: React.ReactNode
+  href: Path
+}
+export const Header = (props: HeaderProps) => {
+  const { title, children, href } = props
+  return (
+    <div className="flex flex-col gap-[16px]">
+      <Link href={href}>
+        <ArrowLeft />
+      </Link>
+
+      <h1 className="text-[24px]">{title}</h1>
+      {children}
+      <div className="relative w-full">
+        <hr className="divider" />
+      </div>
```

Comment:
Not all pages would be like the confirmation page having a big header, so I think these parts can let each page to decide its needs.

Without this part, this component could be renamed to for example `StepBackBar` to reveal its intention.

Reply:
[Rename <Header/> to be <stepBackBar/>](https://github.com/pilagod/waallet-extension/pull/152/commits/e68150764b9c0d755dd3ef0951f4ef844e713b15)